test/distributed/org/apache/cassandra/distributed/test/log/ForceSnapshotTest.java

```diff
@@ -147,6 +148,18 @@ public void testDumpLoadMetadata() throws IOException
                     throw new RuntimeException(e);
                 }
             });
+            long epoch = cluster.get(1).callOnInstance(() -> ClusterMetadata.current().epoch.getEpoch());
+            cluster.forEach(i -> i.runOnInstance(() -> {
```

Comment:
im ok if you want to do this but why not use `org.apache.cassandra.distributed.shared.ClusterUtils#waitForCMSToQuiesce(org.apache.cassandra.distributed.api.ICluster<org.apache.cassandra.distributed.api.IInvokableInstance>, org.apache.cassandra.tcm.Epoch, boolean, int...)`?

with `fetchLogWhenBehind=true` you get the same behavior

Reply:
I did use it, a couple of lines below :) but I should have removed this code of course, thanks!